code/game/machinery/doors/airlock_types.dm

```diff
@@ -318,6 +318,22 @@
 	icon = 'icons/obj/doors/airlocks/cult/runed/cult.dmi'
 	overlays_file = 'icons/obj/doors/airlocks/cult/runed/overlays.dmi'
 	doortype = /obj/structure/door_assembly/door_assembly_cult
+	hackProof = 1
+	aiControlDisabled = 1
+
+/obj/machinery/door/airlock/cult/allowed(mob/M)
+	if(!iscultist(M))
+		PoolOrNew(/obj/effect/overlay/temp/cult/sac, src.loc)
+		var/atom/throwtarget
+		throwtarget = get_edge_target_turf(src, get_dir(src, get_step_away(M, src)))
+		M << pick(sound('sound/hallucinations/turn_around1.ogg',0,1,50), sound('sound/hallucinations/turn_around2.ogg',0,1,50))
```

Comment:
yes, nar-sie is speaking into their mind, though I suppose it would be more useful if EVERYONE nearby heard, althought they would already hear the door rejection noise and see the nar-sie overlay appear (cue cultists streaming out the door to mob the interloper).